scripts/anonymize.sql

```diff
@@ -47,11 +48,14 @@ TRUNCATE tidings_watch;
 
 UPDATE auth_user SET
     -- username left alone, because it's public info
-    password = NULL,
+    password = '!',
     email = CONCAT('user-', id, '@example.com'),
     first_name = ROUND(RAND()*1000000),
     last_name = ROUND(RAND()*1000000);
 
+UPDATE account_emailaddress SET
+    email = CONCAT('user-', id, '@example.com');
```

Comment:
I started with that, but then the records violate a unique constraint, since the same user_id can have multiple account_emailaddress records. E.g., for user ID 8, `user@gmail.com` and `user@yahoo.com` both become `user-8@example.com` which violates a unique key constraint.

So I wasn't sure how else to anonymize the email address. :neutral_face: